README.md

```diff
@@ -1,79 +1,16 @@
 # starknet-contract-verifier
 
-`starknet-contract-verifier` is a contract class verification cli that allows you to verify your starknet classes on a block explorer.
+Client for the [Voyager Starknet block explorer](https://voyager.online), that allows you to verify your starknet classes.
 
-#### The list of the block explorer we currently support are:
-- [Voyager Starknet block explorer](https://voyager.online).
+## Quickstart guide
 
+### Scarb
 
-#### We currently support the following Cairo version & Scarb version.
-- [x] Cairo 1.1.0 & Scarb v0.4.0
-- [x] Cairo 1.1.1 & Scarb v0.4.1
-- [x] Cairo 2.0.1 & Scarb v0.5.1
-- [x] Cairo 2.0.2 & Scarb v0.5.2
-- [x] Cairo 2.1.0 & Scarb v0.6.1
-- [x] Cairo 2.1.1 & Scarb v0.6.2
-- [x] Cairo 2.2.0 & Scarb v0.7.0
-- [x] Cairo & Scarb 2.3.0
-- [x] Cairo & Scarb 2.3.1
-- [x] Cairo & Scarb 2.4.0
-- [x] Cairo & Scarb 2.4.1
-- [x] Cairo & Scarb 2.4.2
-- [x] Cairo & Scarb 2.4.3
-- [x] Cairo & Scarb 2.4.4
-- [x] Cairo & Scarb 2.5.0
-- [x] Cairo & Scarb 2.5.1
-- [x] Cairo & Scarb 2.5.2
-- [x] Cairo & Scarb 2.5.3
-- [x] Cairo & Scarb 2.5.4
-- [x] Cairo & Scarb 2.6.0
-- [x] Cairo & Scarb 2.6.1
-- [x] Cairo & Scarb 2.6.2
-- [x] Cairo & Scarb 2.6.3
-- [x] Cairo 2.6.3 & Scarb 2.6.4
-- [x] Cairo 2.6.4 & Scarb 2.6.5
-- [x] Cairo 2.7.0 & Scarb 2.7.0
-- [x] Cairo 2.7.1 & Scarb 2.7.1
-- [x] Cairo 2.8.0 & Scarb 2.8.0
-- [x] Cairo 2.8.0 & Scarb 2.8.1
-- [x] Cairo 2.8.2 & Scarb 2.8.2
-- [x] Cairo 2.8.2 & Scarb 2.8.3
-- [x] Cairo 2.8.4 & Scarb 2.8.4
+Contract verifier works with [Scarb](https://docs.swmansion.com/scarb) based projects. The tool assumes that `scarb` command is available in the envirenment and project is building properly by executing `scarb bulid`.
```

Comment:
It would be useful to know which versions are currently supported by the backend, or at least what is the latest supported version